vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/components/scrollview/ScrollViewProps.cpp

```diff
@@ -117,9 +117,7 @@ ScrollViewProps::ScrollViewProps(
                     rawProps,
                     "endDraggingSensitivityMultiplier",
                     sourceProps.endDraggingSensitivityMultiplier,
```

Comment:
Do we still need this file or can we delete it?

Reply:
I also see that NativeAnimatedHelper.win32.js is still in the repo, I'm not sure https://github.com/microsoft/react-native-windows/issues/11041 is actually resolved

Reply:
Removed #11041 from resolved issues. Deleted the unused fork.